scripts/build_in_container.sh

```
@@ -0,0 +1,179 @@
+# plugins ctypes-load libnccl.so.2 / libnixl.so from these wheels at first
+# use.
+uv pip install --python "${VENV}/bin/python" --no-deps \
+    "nixl-cu13>=1.0.1" \
+    "nvidia-nccl-cu13>=2.30.4"
+
+# ---- 6. FlashInfer + both EP backends -------------------------------------
+# Wipe any half-populated meson subproject extracts from prior aborted runs.
+# Meson refuses to setup a subproject whose dir exists but lacks a
+# meson.build (happens when the tarball extract from packagecache was
+# interrupted). Wraps + packagecache are kept so re-extract is fast.
+for sp in /tmp /var/tmp "${REPO_ROOT}/3rdparty/nixl/subprojects"; do : ; done
```

Shellcheck (0.11.0)

[warning] sp appears unused. Verify use (or export if used externally).

(SC2034)